js/controllers/modal/ModalHandler.js

```diff
@@ -192,6 +192,9 @@
 	 */
 	$.pkp.controllers.modal.ModalHandler.prototype.modalOpen =
 			function($handledElement) {
+		if (!$handledElement) {
```

Comment:
Watch out for space-based indents. Is this to correct a bug or prevent a warning or something?

Reply:
Honnestly I don't know why it has been added here. I will remove it for now, but gonna keep an eye on it in case of a possible exception.